ext/js/display/display.js

```diff
@@ -1112,6 +1112,17 @@ class Display extends EventDispatcher {
                 }
                 button.disabled = !canAdd;
                 button.hidden = false;
+
+                if (button.disabled && tags && tags.length >= 1) {
```

Comment:
Based on comments in backend.js, the settings checks would go here; use `this._options`. Would also recommend just putting this new section of logic into a private function like `this._setupTagsIndicator(i, noteInfos);`

For the `if` statement:
* `if (Array.isArray(tags) && tags.length > 0) {`
* (`disabled` check likely unnecessary)